python-avocado.spec

```diff
@@ -175,7 +175,10 @@ popd
 pushd optional_plugins/resultsdb
 %{__python} setup.py develop --user
 popd
-selftests/run
+# Package build environments have the least amount of resources
+# we have observed so far.  Let's avoid tests that require too
+# much resources or are time sensitive
+AVOCADO_CHECK_LEVEL=0 selftests/run
```

Comment:
OK, please ignore this and next time rebase it with `--ignore-date`. Github is confused....